sigopt/hyperopt/run_builder.py

```diff
@@ -0,0 +1,34 @@
+from ..run_context import BaseRunContext
+import copy
+
+class RunBuilder(BaseRunContext):
```

Comment:
I think `HyperOptRunContext` makes more sense

Reply:
I try to keep RunBuilder in common, other integration can use it.

Reply:
then you should place this on the `sigopt-python`. The word builder is also misleading.

Reply:
It is used to build a dict for a run.  Name changed, and moved to `sigopt-python`.